packages/verification-vue-component/src/workers/pdf.worker.js

```diff
@@ -0,0 +1,21 @@
+import PdfWasm from '../wasm/pdf.wasm.init'
+
+self.addEventListener('message', async (e) => {
+    try {
+        console.log("pdfwasm init")
+        await PdfWasm.init()
+
+        const pdfBytes = self.wasmProcessPdf(e.data.pdfBytes, e.data.watermarkPdfBytes, e.data.watermarkReplacements)
+
+        self.postMessage({
+            status: true,
+            pdfBytes,
+        })
+    } catch (error) {
+        self.postMessage({status: false, error: 'file encrypted'})
+    }
+}, false)
+
+export default {
+
+}
```

Comment:
is this pdf.worker.js used somewhere? The only import of it i found was commented.